### DISCUSSION TITLE
Get repo branch fails with "Resource not accessible by integration" error

### DISCUSSION THREAD

**Discussion by denar90** [octokit/octokit.js#2065] [Q&A]
Howdy folks 👋 

We have auth process in our website. So user can authenticate with Github, hence they provide us a token we can use to work with their repos, etc.

The issue I faced related to fetching particular branch from repo isn't related to the user who's related to the token.
https://docs.github.com/en/rest/reference/repos#list-branches--code-samples

When I use user token and call github API like `/repos/{owner}/{repo}/branches` it returns me info about branches.
When I want to call `/repos/{owner}/{repo}/branches/{branch}` it fails with error `Resource not accessible by integration`.

In case of fetching branch `/repos/{owner}/{repo}/branches/{branch}` of related repo to the user - it works fine.

Has anyone faced similar issues or can know what happens? Thanks in advance 😃

**Comment by gr2m** [comment]
> Resource not accessible by integration

That error message implies that you are authenticating as a GitHub App installation, not as a user. You don't seem to be using the token provided by the user for this request
